COLE-readthis.md

```diff
@@ -0,0 +1,38 @@
+* Notes
+** I wasn't sure where to put my comments about steps/tasks/instructions in the code because everything is spread out over different files. If I put notes in one file you might not know whether I went there first or second or whatever. So here it is.
+** After reading the README 3 or 4 times I had no idea what I started to get a little upset because I wasn't sure what half of it meant, so I used the checklist that Ellen Nitchals sent out to guide me. Sorry. I know I'm supposed to follow the README but half the time they just confuse me more than I already was and I just go by the checklists as they make a lot more sense and are much easier to follow.
+** Question: single quotes or double quotes?
+
+* Project Organization
+
+** Step 1: npm setup
+*** I understood this part of the README and it's just like a lot of other projects we've done recently so everything went according to plan except for the last part, which says that I should get an error message. I did not. When I ran npm start in my village, I saw input boxes for name, age, and height, an "Add to the village" button, and "Smurf Village" written beneath. There was no error, it was just empty.
+
+** Step 2: tell axios to get the smurfs that are already on the server
+*** I did this in my App.js document. The instructions already commented in there were kind of confusing to me. I'm not sure what they meant by passing the right properties on state down to props so I ignored this stuff and eventually deleted it because it was confusing me.
```

Comment:
Generally if you are going to pass something to your server. Passing some state down to props, in this case, would be the name, age, and height most likely. 

```
<Smurf height={this.state.height} age={this.state.age} name={this.state.name}/>
```
Now when you are in the `Smurf` component. You can call `props.height` or `props.name` or `props.age` accordingly. Thus, passing state down to props

Reply:
Hope that makes sense